store/src/java/com/zimbra/cs/ephemeral/EphemeralStore.java

```diff
@@ -234,31 +246,41 @@ public static void setAutoloadExtensions(boolean bool) {
         autoloadExtensions = bool;
     }
 
-    private static String getFactoryClassName(String backendName) throws ServiceException {
```

Comment:
I don't understand the logic for having a static autoloadExtensions field and then an overriding parameter disableAutoload. Also, why do we need to try and find again - what may change after the application tries loading extensions and before this code executes?